server/model/monitor.js

```diff
@@ -891,6 +893,12 @@ class Monitor extends BeanModel {
                     bean.status = UP;
                     bean.ping = dayjs().valueOf() - startTime;
 
+                } else if (this.type === "zookeeper") {
```

Comment:
We have recently started using a better syntax for creating monitors.
It is documented in our contribution guide here:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#:~:text=new%20monitoring%20types

The currently migrated montiors live here:
https://github.com/louislam/uptime-kuma/tree/master/server/monitor-types

All new monitors MUST to use this, to use this format instead.
Rationale: We can't keep dumping all code in the giormous `monitor.js`. This does not scale in terms of maintenance/.. effort.